lib/tcpip/driver_adaptor.cc

```diff
@@ -68,6 +68,10 @@ namespace
 bool __cheri_compartment("TCPIP")
   ethernet_receive_frame(uint8_t *frame, size_t length)
 {
+	if (!check_pointer<PermissionSet{Permission::Load}>(frame, length))
```

Comment:
OK, then I will a comment there to clarify that.

Reply:
It would be nice to reference [this line](https://github.com/CHERIoT-Platform/network-stack/blob/9e327b7f14279e1189c4fbfc8dd404f683c30a74/network_stack.rego#L101
) in the policy file:

```rego
	data.compartment.compartment_call_allow_list("Firewall", "ethernet_send_frame.*", {"TCPIP"})
```

Reply:
Sounds good, will do.